README.org

```diff
@@ -53,18 +53,16 @@
    - 🌟 *Flexible* - could be configured as full-blown IDE with flashy UI or minimal distraction free
 ** Overview
    Client for [[https://github.com/Microsoft/language-server-protocol/][Language Server Protocol]] (v3.14). [[https://github.com/emacs-lsp/lsp-mode][lsp-mode]] aims to provide IDE-like experience by providing optional integration with the most popular Emacs packages like ~company~, ~flycheck~ and ~projectile~.
-
    - Non-blocking asynchronous calls
-   - Real-time Diagnostics/linting (via builtin ~flymake~ when Emacs > 26, requires flymake>=1.0.5 or [[https://github.com/flycheck/flycheck][flycheck]]/[[https://github.com/emacs-lsp/lsp-ui][lsp-ui]])
-   - Code completion - using [[https://github.com/tigersoldier/company-lsp][company-lsp]] or builtin ~completion-at-point~
+   - Real-time Diagnostics/linting (via ~flymake~ when Emacs > 26, requires flymake>=1.0.5 or builtin [[https://github.com/flycheck/flycheck][flycheck]])
```

Comment:
It is flymake not flycheck which is builtin?

Reply:
Flymake is builtin and flycheck is external